packages/addresses/src/addresses.helpers.ts

```diff
@@ -1,9 +1,8 @@
-import dotenv from "dotenv";
-import { readdirSync, readFileSync, writeFileSync } from "fs";
 import { network } from "hardhat";
-import { join } from "path";
 
-import { deepCloneSerializable } from "./misc.helpers";
+import athens from "./addresses.athens.json";
+import mainnet from "./addresses.mainnet.json";
+import troy from "./addresses.troy.json";
```

Comment:
Can we use dynamic imports or require in the function itself for these? The issue of importing them statically is that they get cached and that breaks our deploy scripts (when you have a script that runs deploy A, B and C, and deploy B depends on an address that was set in deploy A).

Reply:
added a new method getAddressAsync to reload async if needed